metricq_source_ipmi/plugin_power_supply/plugin.py

```diff
@@ -0,0 +1,25 @@
+import logging
+import logging.handlers
+import click_log
+from metricq.logging import get_logger
+
+logger = get_logger()
+
+click_log.basic_config(logger)
+sh = logging.handlers.SysLogHandler()
+logger.addHandler(sh)
+logger.setLevel('INFO')
+logger.handlers[0].formatter = logging.Formatter(
+    fmt='%(asctime)s [%(levelname)-8s] [%(name)-20s] %(message)s')
+
+NEEDED_SENSORS = 4
+POWER_SUPPLY_STATUS_OK = ["'OK'", "'Presence detected'"]
+
+def create_metric_value(sensors):
+    logger.info(f"sensors: {sensors}")
```

Comment:
logging every input seems a bit excessive to me.

Reply:
You are right, it was just for debugging. I will remove the line.